src/app/page.tsx

```diff
@@ -127,110 +144,116 @@ export default function Home() {
+        <div className="text-left text-xl ml-5 md:mt-36 md:mb-10 break-words md:col-span-5 md:ml-20">
           An interactive exhibition in Paris to celebrate Creativity, Diversity, and Unity.
         </div>
-        <div className="hidden md:block md:col-span-6"></div>
-        <div className="hidden md:block md:col-span-1"></div>
-        <div className="md:col-span-3 bg-neutral-white rounded-r-full rounded-l-2xl md:rounded-2xl shadow-md flex flex-row md:flex-col items-center md:ml-20 ml-5 mr-5">
-          <Image
-            src="/home/icon1.svg"
-            width={86}
-            height={86}
-            className="hidden md:block md:w-3/7 w-1/2 h-auto md:mt-3"
-            alt="Painting panel with mixed paints"
-          />
-          <div className="break-words text-base text-center md:pt-7 md:pr-7 md:pl-7">
-            Visitors ages 10 to 20 Create, Upload, and Share their masterpieces to win the Gold!
+      </div>
+      <div className="col-span-1 md:col-span-10 flex justify-center bg-secondary-blue pt-10 md:pt-0">
+        <div className="md:w-4/5 flex flex-col md:flex-row">
+          <div className="md:w-1/3 bg-neutral-white rounded-r-[6rem] rounded-l-[2rem] md:rounded-2xl p-7 mb-6 md:mb-0 ml-5 mr-5 shadow-md flex flex-row md:flex-col items-center">
+            <Image
+              src="/home/icon1.png"
+              width={86}
+              height={86}
+              className="hidden md:block md:mt-3"
+              alt="Painting panel with mixed paints"
+            />
+            <div className="break-words text-base md:text-center text-start md:pt-7 md:px-0">
+              Visitors ages 10 to 20 Create, Upload, and Share their masterpieces to win the Gold!
+            </div>
+            <Image
+              src="/home/icon1.png"
+              width={86}
+              height={86}
+              className="block md:hidden"
+              alt="Painting panel with mixed paints"
+            />
           </div>
-          <Image
-            src="/home/icon1.svg"
-            width={86}
-            height={86}
-            className="block md:hidden md:w-3/7 w-1/2 h-auto md:mt-3"
-            alt="Painting panel with mixed paints"
-          />
-        </div>
-        <div className="md:col-span-3 bg-neutral-white rounded-l-full md:rounded-2xl p-10 shadow-md flex flex-row md:flex-col items-center ml-5 mr-5 md:ml-10 md:mr-10">
-          <Image
-            src="/home/icon2.svg"
-            width={86}
-            height={86}
-            className="w-1/4 md:w-1/2 md:mt-3"
-            alt="Written document with a checkmark"
-          />
-          <div className="break-words text-base text-center md:pt-10">
-            Visitors vote for their favorite artwork.
+          <div className="md:w-1/3 bg-neutral-white rounded-l-[6rem] rounded-r-[2rem] md:rounded-2xl p-6 shadow-md flex flex-row md:flex-col items-center mb-6 md:mb-0 ml-5 mr-5 md:ml-2">
```

Comment:
Please make sure each of these cards have the exact same classes (other than for the border radius). I'm noticing some discrepancies